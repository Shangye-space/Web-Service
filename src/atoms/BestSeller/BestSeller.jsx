import React from 'react';
import classnames from 'classnames';

import styles from './BestSeller.module.scss';

const BestSeller = () => {
    return (
        <div className={classnames('col-6', styles.bestSellerWrapper)}>
            <img src="/images/989171832_o.jpg" alt="" />
        </div>
    );
};

export default BestSeller;
