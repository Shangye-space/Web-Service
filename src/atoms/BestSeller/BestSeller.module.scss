@import '../../settings/variables.scss';

.bestSellerWrapper {
    width: 100%;
    height: 510px;

    padding: 0;

    img {
        height: 100%;
        width: 100%;
        padding: 0 0;
        border-radius: 2%;

        filter: brightness(85%);
    }
}
