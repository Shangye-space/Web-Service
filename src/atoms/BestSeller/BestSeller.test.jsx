import React from 'react';
import ReactDOM from 'react-dom';
import BestSeller from './BestSeller.jsx';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BestSeller />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('snapshot renders', () => {
    const component = renderer.create(<BestSeller />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
