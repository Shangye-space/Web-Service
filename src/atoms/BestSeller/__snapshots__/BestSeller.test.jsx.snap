// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot renders 1`] = `
<div
  className="col-6 bestSellerWrapper"
>
  <img
    alt=""
    src="/images/989171832_o.jpg"
  />
</div>
`;
